### ISSUE TITLE
Existential Quantification reordered

### ISSUE THREAD

**Issue by DavidEichmann** [commercialhaskell/hindent#443]
The existentially quantified types keep getting reversed:

Input
```haskell
data D =
  forall a b c. D a
                  b
                  c
```
Output
```haskell
data D =
  forall c b a. D a
                  b
                  c
```
Output (after calling hindent on the above output)
```haskell
data D =
  forall a b c. D a
                  b
                  c
```

I suspect this is an easy fix.

**Comment by DavidEichmann** [comment]
Is this related to https://github.com/haskell-suite/haskell-src-exts/issues/328?

**Comment by sighingnow** [comment]
Which version of hindent do you use and which version of haskell-src-exts do you use to build hindent ?

**Comment by sighingnow** [comment]
It's seems a regression.

**Comment by DavidEichmann** [comment]
```
$ stack exec -- hindent --version
hindent 5.2.3
```

I'm not sure how to get the version of haskell-src-exts, but I tried this:

```
stack exec -- ghc-pkg list
haskell-src-exts-1.18.2
```

**Comment by sighingnow** [comment]
Close as fixed in commit 4556ff9.
